tasks/main.yml

```diff
@@ -21,6 +21,7 @@
   set_fact:
     timesync_services: "{{ ansible_facts.services.values() |
           selectattr('name', 'defined') |
+          rejectattr('status', 'undefined') |
```

Comment:
I don't think `undefined` is a valid test - I think you should use this instead
`selectattr('status', 'defined') |`

Reply:
Fair enough, I had that first time around (both worked). This is cleaner :)